bigquery_etl/cli/query.py

```diff
@@ -457,14 +458,22 @@ def _backfill_query(
     dry_run,
     no_partition,
     args,
+    partitioning_type,
     backfill_date,
 ):
     """Run a query backfill for a specific date."""
     project, dataset, table = extract_from_query_path(query_file_path)
 
+    match partitioning_type:
+        case PartitionType.DAY:
+            partition = backfill_date.strftime("%Y%m%d")
+        case PartitionType.MONTH:
+            partition = backfill_date.strftime("%Y%m")
+        case _:
+            raise ValueError(f"Unsupported partitioning type: {partitioning_type}")
+
```

Comment:
Not a big deal, but this could go into the `if` block below (line 477).

Reply:
Thank you. Although it will probably make it perform better, I'll leave it as is because below that `if` block, `backfill_date` is already a string in `"%Y-%m-%d"` format and I think working around that will add to code complexity.